### PR TITLE
Update header datetime

### PR DESCRIPTION
**Pull request by frankfu03**

### CODE REVIEW

**Comment by lbodor** [comment]
:+1:

File: src/client/app/shared/abstract-groups-items/abstract-item.component.ts [hunk 1]
```diff
@@ -217,9 +217,8 @@ export abstract class AbstractItemComponent extends AbstractBaseComponent implem
      */
     public getItemHeaderHtml(): string {
 
-        // TODO - change view model to be exclusively Date and fix this
-        let startDatetime: any = this.getItem().startDate;
-        let endDatetime: any = this.getItem().endDate;
+        let startDatetime: any = this.itemGroup.controls.startDate.value;
```
Comment (oehm-smith) [inline src/client/app/shared/abstract-groups-items/abstract-item.component.ts]: Wouldn't that be `this.getItem().startDate.value`?

Comment (frankfu03) [inline src/client/app/shared/abstract-groups-items/abstract-item.component.ts]: No. "this.getItem()" return a view model and "startDate" is one of the fields in the view model. It does not have "value" field.

**Comment by lbodor** [comment]
Please rebase to resolve conflicts.

**Pull request merged**